qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -40,6 +40,21 @@
     nic_model_nic1 = rtl8139
     q35:
         nic_model_nic1 = e1000e
+    # install winfsp tool
+    i386, i686:
+        install_path = 'C:\Program Files'
+    x86_64:
+        install_path = 'C:\Program Files (x86)'
+    install_cmd = 'msiexec /i WIN_UTILS:\winfsp-1.9.21096.msi /qn'
+    check_installed_cmd = 'dir "%s" |findstr /I winfsp'
+    viofs_sc_create_cmd = 'sc create VirtioFsSvc binpath="%s" start=auto'
+    viofs_sc_create_cmd += ' depend="WinFsp.Launcher/VirtioFsDrv" DisplayName="Virtio FS Service"'
+    viofs_sc_start_cmd = 'sc start VirtioFsSvc'
+    viofs_sc_query_cmd = 'sc query VirtioFsSvc'
+    viofs_sc_delete_cmd = 'sc delete VirtioFsSvc'
+    viofs_sc_stop_cmd = 'sc stop VirtioFsSvc'
```

Comment:
for these cmds, is there often changed for different test cases? if it's not, in provider, we may use it like this:
start_cmd = params.get('viofs_sc_start_cmd', 'sc start VirtioFsSvc')
then we don't have to define these params for each case unless one day we have to use different options, then we define it explicitly in cfg